tests/phpunit/class-wc-stripe-intent-controller-test.php

```
@@ -120,6 +120,74 @@ public function test_automatic_capture_from_the_settings() {
+					[
+						'id'            => 1,
+						'client_secret' => '123',
+					]
+				),
+			];
+		};
+
+		add_filter( 'pre_http_request', $test_request, 10, 3 );
+
+		$this->mock_controller->create_payment_intent( $this->order->get_id() );
+	}
```

PHPMD (2.15.0)

[error] The method test_create_payment_intent_uses_order_currency is not named in camelCase. (undefined)

(CamelCaseMethodName)